### ISSUE TITLE
just show left half label

### ISSUE THREAD

**Issue by wuqiong4945** [tiagofumo/vim-nerdtree-syntax-highlight#8]
ubuntu, vim 7.4

**Comment by wuqiong4945** [comment]
![screenshot from 2016-09-23 23-30-50](https://cloud.githubusercontent.com/assets/1294412/18791580/e8f0e7ee-81e5-11e6-8ed1-7b85abd3ddfc.png)

**Comment by tiagofumo** [comment]
If you disable the plugin and use  vim-devicons without colors, does it work?

**Comment by ryanoasis** [comment]
I was going to say this is related to ryanoasis/vim-devicons/issues/133 but I don't know if it is quite the same (some are not cut and only in the tree and this is on Ubuntu not Mac)

**Comment by wuqiong4945** [comment]
turn out the plugin and set color by manual. It is all OK.

![screenshot from 2016-09-24 21-27-21](https://cloud.githubusercontent.com/assets/1294412/18808764/baea0a5c-829d-11e6-87ab-fcd7bd35812a.png)

**Comment by tiagofumo** [comment]
@wuqiong4945 in that picture the python icon and the lua icon are cuted. I think there is something wrong with your setup. Comment the lines to manually set the colors and check if the icons are ok. If not, try another patched font, because some of these icons are just too big.

**Comment by xusiyuan841028** [comment]
I have tried to fix the bug, FYI.
https://github.com/xusiyuan841028/vim-nerdtree-syntax-highlight/commit/9b35c3ea2c70b2ba629609e65e8741b275a18f16

**Comment by audacioustux** [comment]
having same problem on neovim 0.2.0-dev
emulator: Konsole
font: Droid sans for powerline nerd font
added `autocmd FileType nerdtree setlocal nolist` but still not working...

**Comment by felixsanz** [comment]
I can reproduce the error only when this plugin activated. Vimdevicons works fine without this.

**Comment by magler** [comment]
In MacVim  8.0.596 I unchecked `Prefrences->Advanced-> Use Core Text renderer`

**Comment by IsOverThere** [comment]
same issue with VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Jul 12 2017 07:40:16) on 4.10.0-32-generic #36~16.04.1-Ubuntu SMP Wed Aug 9 09:19:02 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

Half icons issue can be reproduced only when vim-nerdtree-syntax-highlight is enabled. When I removed vim-nerdtree-syntax-highlight, no problem for the show of the icons.

**Comment by purejoy** [comment]
I have the same issue with NeoVim 0.3.0 + URxvt + Meslo LGM Nerd Font on Gentoo Linux.  When deactivating this plugin, vim-devicons works fine. 
![1](https://user-images.githubusercontent.com/3711998/44341467-89f1d600-a4ba-11e8-95e0-7b86b11b13aa.png)
![2](https://user-images.githubusercontent.com/3711998/44341468-8b230300-a4ba-11e8-94f5-d0b3c60abb4d.png)
![4](https://user-images.githubusercontent.com/3711998/44341596-ed7c0380-a4ba-11e8-999a-272273f90b1e.png)

I've tried some other nerd fonts like DroidSans, Hack, and Dejavu, but  it's still rotten luck.

**Comment by andys8** [comment]
Same issue for me

![image](https://user-images.githubusercontent.com/13085980/69011924-1d40e200-0970-11ea-8bc3-92e523f66e13.png)

(Vim 8.1, Suckless Terminal, Iosevka Nerd Font)

**Comment by allbombson** [comment]
Works fine for me with nvim and The offical IBM Plex nerdfonts patch

**Comment by xjantoth** [comment]
Same problem with half icons in vim 8.2

**Comment by purejoy** [comment]
This issue can still be reproduced on rxvt-unicode. I found a workaround to solve this.
![2021-06-01_16-04](https://user-images.githubusercontent.com/3711998/120288727-1fb92180-c2f3-11eb-9886-e134c05905eb.png)
[fix_missing_right_side.patch.txt](https://github.com/tiagofumo/vim-nerdtree-syntax-highlight/files/6574540/fix_missing_right_side.patch.txt)

**Comment by Leo310** [comment]
The font: "NotoMono Nerd Font Mono" worked for me on the Konsole. 
![image](https://user-images.githubusercontent.com/48623649/133325597-30ee3dd3-650a-4ef4-bbcd-33501e43bce9.png)

**Comment by ToppDev** [comment]
Hello, I still have this issue

**Comment by ToppDev** [comment]
Hello, I also have this issue.

Only with vim-devicons without highlighting I don't have the cut-off.

As other people said, it can be fixed by using fonts. Which really are monospaced and therefore having smaller icons which fit into one character. However I don't see switching fonts as a valid solution to this problem.
![2022-04-20_06-39](https://user-images.githubusercontent.com/17766093/164151985-6ae7751f-7634-428d-bf7f-04234cb6f08f.png)

My setup:
- Terminal: Suckless Terminal with a few patches (https://github.com/ToppDev/st)
- Nerd Fonts: https://aur.archlinux.org/packages/nerd-fonts-complete
- Font with cut-off: FiraCode Nerd Font
- Font without cut-off: FiraCode Nerd Font Mono

**Comment by jclsn** [comment]
Another problem is that all the truly Mono Nerd Font versions don't work with Powerline10k. I am ditching this plugin until this is fixed.

**Comment by jclsn** [comment]
So I guess this plugin is abandoned. Pity...

**Comment by Leo310** [comment]
@jclsn just use nvim-tree. It's way better

**Comment by jclsn** [comment]
@Leo310 I don't use Neovim

**Comment by Leo310** [comment]
@jclsn then switch to neovim :p it's also way better

**Comment by jclsn** [comment]
No, I reject Neovim. I don't like the mindset behind it. [Fern ](https://github.com/lambdalisue/fern.vim) seems like a good alternative to NERDTree
